tools/wpt/browser.py

```diff
@@ -789,7 +789,7 @@ def _get_webdriver_url(self, version):
         # Make sure we use the same revision in an invocation.
         # If we have a url that was last used successfully during this run,
         # that url takes priority over trying to form another.
-        if self.last_url_used is not None:
+        if hasattr(self, "last_url_used") and self.last_url_used is not None:
```

Comment:
```suggestion
        if hasattr(self, "last_url_used"):
```
We should be able to remove the second logic statement here, as `hasattr()` will return false if the attribute exists but is null.